apps/examples/webserver_test/readme.md

```diff
@@ -0,0 +1,48 @@
+# Webserver Test Application
+
+### Features
+
+- HTTP/HTTPS
+- GET/PUT/POST/DELETE method
+- Supports Keep-alive
+- Chunked Transfer
+- Trailer header
+
+### Menuconfig Setup
+**For build configuration details refer to**
+'build/configs/rtl8721csm/security_hal_test_tz/defconfig'
+
+**In SE Selection turn-off HW rsa verification and HW rsa encryption to test in ecdsa**
+
+**Enable web server**
+> Networking support > Protocols > Webserver 
+
+> Application Configuration > Examples > Webserver example
+
+> Application Configuration > Examples > Webserver test
+
+## webserver_test usage :
+
+**webserver_test iterations method uri [options...]**
+
+- iterations range [1,10000]
+- method and uri parameters should be given,  [options] parameters are optional.
+- method is one of "GET", "POST", "PUT", "DELETE".  
+- url should start with "http://" or "https://".
+   
+[options] parameter support several different options.
+-    chunk-count=Z        : Enables chunked encoding and send Z chunks (default is disabled)
+-    test_entity=SIZE : Input test entity dummy data (default is 0)
+-    keep-alive=X     : Turns on keep-alive and sends X requests (default Keep-Alive is OFF)
+-    trailer=1        : Turns on Trailer-header
+
+Example usage:
+`webserver_test 3 POST https://127.0.0.1/ chunk-count=4 test_entity=5 trailer=1 keep-alive=2`
+
```

Comment:
I input above command but it failed. Do I miss something please check.
```
TASH>>webserver_test 3 POST http://127.0.0.1 chunk-count=4 test_entity=5 trailer=1 keep-alive=2

TASH>>[WS_TEST]webserver_test_main():918: entry

[WS_TEST]ws_test_prepare_request():810: entry

[WS_TEST]ws_test_prepare_request():883: exit

[WS_TEST]ws_test_process_query():892: entry

[WS_TEST]ws_test_http_client_response_init():88: entry

[WS_TEST]ws_test_http_client_response_init():111: exit

[WS_TEST]ws_test_http_client_send_request():739: entry

[WS_TEST]ws_test_base():536: entry

[WS_TEST]ws_test_parsehttpurl():487: entry

[WS_TEST]ws_test_parsehttpurl():530: exit

[WS_TEST]ws_test_base():571: hostname='127.0.0.1'

[WS_TEST]ws_test_msg_construct():147: entry

[WS_TEST]ws_test_chunksize():68: entry

[WS_TEST]ws_test_chunksize():82: exit

[WS_TEST]ws_test_msg_construct():353: exit

[WS_TEST]ws_test_base():590: sndlen:229 , param->buffer is:

POST / HTTP/1.1

host: 127.0.0.1

Content-Type: application/x-www-form-urlencoded

Transfer-Encoding: chunked

Trailer: Expires

Connection: keep-alive

User-Agent: TinyARA



5

11111

0

Expires: Fri, 21 Oct 2022 07:28:00 GMT



[WS_TEST]ws_test_socket_connect():359: entry

[WS_TEST]ws_test_socket_connect():373: recv timeout is set to (50.0)sec

[WS_TEST]ws_test_gethostip():129: entry

[WS_TEST]ws_test_gethostip():141: exit

[WS_TEST]ws_test_socket_connect():398: ERROR: connect failed: -1 errno: 104

[WS_TEST]ws_test_base():594: ERROR: socket failed: 0

[WS_TEST]ws_test_base():731: exit
```

Reply:
Did you enable webserver? If not enable it using `webserver start none`